### DISCUSSION TITLE
Would it be possible to get a release with the FixWinSdkIn41 included?

### DISCUSSION THREAD

**Discussion by cmonachan-git** [Mapsui/Mapsui#2770] [Q&A]
I saw that the issue with crashing on start-up has been fixed in the main branch and the 4.1 branch, would it be possible to get a release featuring this fix? 

I'm working on a project that requires this component and if possible I'd like to just pull in an update through the nuget package manger and avoid having to build everything myself.

Apologies in advance, if there's already a way of me easily doing that, and I've missed it.

**Comment by inforithmics** [comment]
One solution is to explicitly install SkiaSharp [2.88.9-preview.2.2](https://www.nuget.org/packages/SkiaSharp/2.88.9-preview.2.2)

**Comment by pauldendulk** [comment]
Mapsui 4.1.8 was released to fix this: https://github.com/Mapsui/Mapsui/releases/tag/4.1.8
